### DISCUSSION TITLE
3ο παραδοτέο

### DISCUSSION THREAD

**Discussion by MariaSiafali117** [merkourisa/Augmented-Reality#24] [Q&A]
Καλησπέρα, στο τρίτο παραδοτέο όταν αλλάζω τις συντεταγμένες και βάζω αυτές που θέλω δεν μου παίζει το animation, ούτε το box που υπάρχει ήδη. Με τις αρχικές συντεταγμένες παίζει κανονικά.

**Comment by merkourisa** [comment]
Καλησπέρα, οι ερωτήσεις που αφορούν στο 3ο παραδοτέο πρέπει να γίνονται στο αντίστοιχο [issue](https://github.com/merkourisa/Augmented-Reality/issues/3). 

Δεν αρκεί να αλλάζει τις συντεταγμένες πρέπει να αλλάξεις και τις συντεταγμένες της κάμερας
